testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/AcquireTokenFragment.java

```diff
@@ -180,7 +180,7 @@ public void onClick(View v) {
         mDebugBrokers.setTextOn("Debug Brokers");
         mDebugBrokers.setChecked(BrokerData.getShouldTrustDebugBrokers());
 
-        mCache = ClientActiveBrokerCache.Companion.getCache(
+        mCache = ClientActiveBrokerCache.getClientSdkCache(
```

Comment:
would it not be ClientActiveBrokerCache.Companion.getClientSdkCache ? given that the new methods are also inside Companion  block?

Reply:
Any companion method marked with JvmStatic doesn't need companion when invoked.

(In kotlin, companion ~= static)